### ISSUE TITLE
Implement command to get random movie from list

### ISSUE THREAD

**Issue by qaamos** [vilukissa68/discord_movie_bot#3]

**Comment by vilukissa68** [comment]
Tätä voisi lähestyä kahdella tavalla

1.  Uusi funktio db.rs tiedostoon, joka pyytää satunnaisen rivin `SQL SELECT RANDOM` tyylisellä queryllä
2.  Implementaatio suoraa uuteen komentoon, jossa hyödynnetään jo olemassa olevaan `db::get_movies()` funktiota ja valitaan satunnaisesti tämän paluuarvosta rivi.

Vaihtoehto 1 on tehokkaampi, mutta vaatii uuden query funktion kirjoittamista. Vaihtoehto 2 on suoraviivaisempi ja nopeampi toteuttaa.

**Comment by qaamos** [comment]
En oo varma voiko tapaa 1 toteuttaa järkevästi / niin että se ois tehokkaampi.

**Comment by qaamos** [comment]
En oo varma pystyykö vaihtoehtoa 1 toteuttamaan tehokkaasti. Käsittääkseni satunnaisen rivin valitseminen SQL:ssä tapahtuu valitsemalla ensin kaikki rivit ja ottamalla sitten niistä yksi satunnaisesti. Eli sama tehokkuus kuin olemassaolevan funktion käyttäminen.

**Comment by vilukissa68** [comment]
Sen pitäis kuitenkin olla nopeampaa tehdä suoraan SQL:ssä kuin siirtämällä muistia ohjelman sisällä. Tosin tässä sillä ei oo kauheesti merkitystä.

**Comment by qaamos** [comment]
Totta, onhan se muistin käytön kannalta parempi. Vois yrittää ensin tavalla 1 ja jos menee turhan työlääksi niin sitten tavalla 2.
